napari/layers/image/image.py

```diff
@@ -536,7 +536,7 @@ def interpolation(self):
         """
         warnings.warn(
             trans._(
-                "Interpolation attribute is deprecated. Please use interpolation2d or interpolation3d",
+                "Interpolation attribute is deprecated since 0.4.16. Please use interpolation2d or interpolation3d",
```

Comment:
I think these ones will be since the version of the next release (maybe 0.4.17? or will it be 0.5?). Not quite sure how we want to handle these.

Reply:
I guess you could say `after` instead of `since`, but not great for consistency.

Reply:
yes. It is introduced after the 0.4.16 release.

Reply:
```suggestion
                "Interpolation attribute is deprecated since 0.4.17. Please use interpolation2d or interpolation3d",
```

My bad I got confused in my git blame. One more reason to add these values in the same PR the deprecations messages are introduced.